### ISSUE TITLE
Add a wiki: Xeno Series Wiki

### ISSUE THREAD

**Issue by KevinPayravi** [KevinPayravi/indie-wiki-buddy#46]
An anonymous user has submitted a new wiki via our [submission form](https://form.getindie.wiki).

Link to origin wiki: https://xenoblade.fandom.com/wiki/Xenoblade_Wiki
Link to destination (independent) wiki: https://www.xenoserieswiki.org/wiki/Main_Page

**Comment by KevinPayravi** [comment]
The independent Xeno Series Wiki originally wasn't included due to the large content gap. It's been re-submitted by a user, so it will be re-reviewed.

**Comment by Elaeagnifolia** [comment]
24,000 vs. 18,400 content pages. 

While there are some things that the independent Wiki does well (Cleaner presentation, utilization of maps for locating, automation of datamined content), it does appear there are pages on the Fandom Wiki that go more in-depth or do not yet exist on the independent Wiki (e.g. Details on XC3 boss fights, story/lore). In their current states, it seems users may still need to go to one site or the other to find information they need.

I am not overly familiar with the Xeno Series though, so anyone more familiar with the subject matter can still chime in if desired.

**Comment by cocona20xx** [comment]
Yeah, that is accurate sadly—the Xeno Series Wiki is somewhat incomplete, primarily due to it's far larger scope than the Xenoblade or Xenosaga (+ Xenogears) Wikia Wikis, as it covers all games in the metaseries. Furthermore, it's a far younger wiki than either of the wikia ones for the metaseries, having been created in 2020. 

If there was a way to notify people the Xeno Series Wiki exists without strictly redirecting them to it from the Xenoblade and Xenosaga wikis that'd be the best way to handle things for now, probably.

**Comment by SnorlaxMonster** [comment]
In terms of raw number of content pages, the independent Xeno Series Wiki has closed in a lot to the Fandom wiki. It now has [23,492 pages](https://www.xenoserieswiki.org/wiki/Special:Statistics) vs the Fandom Xenoblade Wiki's [24,385 pages](https://xenoblade.fandom.com/wiki/Special:Statistics).

Since the Xeno Series Wiki has a broader scope, this certainly means that the Fandom Xenoblade wiki better covers the Xenoblade franchise. However, now that the content gap seems to have narrowed substantially (despite still being present), perhaps it could make sense to support redirecting.

**Comment by Gamerhenky** [comment]
Bumping this. The Xeno Series Wiki now has more content pages than Fandom Xenoblade Wiki (28,992 vs. 24,426), so we could just add it on IWB as soon as possible.

**Comment by Reggiamoto** [comment]
As an admin on XSW I can say that while we have very good coverage on the gameplay side of things, unfortunately the same cannot really be said for characters yet. Many important characters in the series are still complete stubs on our wiki, unlike the Fandom ones.
Examples:
https://www.xenoserieswiki.org/wiki/Vanea
https://www.xenoserieswiki.org/wiki/Gwin
https://www.xenoserieswiki.org/wiki/Dromarch
https://www.xenoserieswiki.org/wiki/Monica
(okay actually I checked the Fandom wiki and Gwin and Monica are pretty stubby over there too, but still)
These are obviously really important articles, but sadly everyone's focuses have been elsewhere lately. I'd say once we've drastically cut down the amount of these stubs I'd call us "worthy".

**Comment by violine1101** [comment]
I've had a look at the wiki, and it does definitely fit our criteria. Of course there's always room for improvement, but from our side I think we're ready to include the wiki :)
